src/libs/actions/Session/index.ts

```diff
@@ -718,7 +718,7 @@ function requestUnlinkValidationLink() {
     API.write('RequestUnlinkValidationLink', params, {optimisticData, successData, failureData});
 }
 
-function unlinkLogin(accountID: number, validateCode: string) {
+function unlinkLogin(accountID: number | string, validateCode: string) {
```

Comment:
What about the BE side, is it type-agnostic and able to handle both types (number and string, '' or 0 in case of empty id)?

Reply:
I guess it should be able to handle `''` since that's how it was working before, I am not sure about 0 as an empty id, cause 0 can be used for some test purposes, I can try to test it

Reply:
I tested with `''`, `0` and `-1` and had the same result!
![image](https://github.com/Expensify/App/assets/23176449/14cfe4b8-b4cc-4de6-b71f-abc7343c47a1)